LocoAnalogueControlMod/Main.cs

```diff
@@ -79,7 +79,7 @@ static void OnUpdate(UnityModManager.ModEntry mod, float delta)
             if (!listenersSetup)
             {
                 // Gotta wait until we are loaded until registering the listeners
-                if (LoadingScreenManager.IsLoading || !WorldStreamingInit.IsLoaded || !InventoryStartingItems.itemsLoaded) return;
+                //if (LoadingScreenManager.IsLoading || !WorldStreamingInit.IsLoaded || !StartingItemsController.itemsLoaded) return;
```

Comment:
I think StartingItemsController is the right class to use here. Used dnSpy to find that, as you suggested. itemsLoaded is defined as a boolean but isn't recognised here for some reason I don't understand. Commenting this line out didn't stop the mod from working, but I suppose there still needs to be a method to have the mod wait for the game to be loaded. If I understand correctly...